test/scripts/run-e2e-tests.sh

```diff
@@ -93,8 +105,7 @@ cd ${GOPATH}/src/github.com/kubeflow/kfserving
 make deploy-ci
 
 echo "Waiting for KFServing started ..."
-sleep 20
-waiting_pod_running "kfserving-system"
+waiting_for_kfserving_controller
```

Comment:
In my testing, kfserving controller is started, but the webhook is not ready unless waiting for more than 1 minute.